docs/SUMMARY.md

```diff
@@ -26,11 +26,10 @@ http://goshimmer.docs.iota.org/
   - [Object storage](./implementation_design/object_storage.md)
 
 - [Protocol specification](./protocol_specification.md)
-  - [Protocol high level overview](./application_logic/protocol.md)
-  - [Layers](./concepts/layers.md)
-  - [Mana](./specification/001-mana.md)
-  - [Markers](./specification/003-markers.md)
-  - [Glossary](./concepts/glossary.md)
+  - [Protocol high level overview](./protocol_speficiation/protocol.md)
+  - [Components](/protocol_specification/components.md)
```

Comment:
Maybe put mana and marker as sub-document of Components?
like this:
```suggestion
  - [Components](/protocol_specification/components.md)
     - [Mana]()
     - [Markers]()
```

Reply:
The idea was to not list them in the table of content and just make links clearly visible in components until we have more complete set of documents. But I am no longer sure about this approach

Reply:
I have listed them in the overview list.. lets stick to our initial plan of having them in the protocol overview document